src/components/HURUmap/Source/index.js

```diff
@@ -0,0 +1,51 @@
+import { Typography } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import PropTypes from "prop-types";
+import React from "react";
+
+import Link from "@/pesayetu/components/Link";
+
+const useStyles = makeStyles(({ typography, palette }) => ({
+  root: {},
+  name: {
+    fontSize: typography.pxToRem(13),
+    lineHeight: 20 / 13,
+    color: "#666666",
+    display: "inline-flex",
+    fontWeight: 500,
+  },
+  link: {
+    color: palette.text.primary,
+    fontSize: typography.pxToRem(13),
+    lineHeight: 20 / 13,
+    fontFamily: typography.body1.fontFamily,
+    fontWeight: 500,
+  },
+}));
+
+function Source({ name, url, ...props }) {
```

Comment:
Using `children`, `href` instead of `name`, `url` would allow use to use Source exactly as we'd use and Link.
```jsx
 <Source href={link}>Name</Source>
```